INTITA/Modules/Profile/ProfileViewController.swift

```diff
@@ -30,98 +33,87 @@ class ProfileViewController: UITableViewController, Storyboarded {
         tableView.dataSource = self
         tableView.delegate = self
         registerCells()
+        headerViewContent.delegate = coordinator
```

Comment:
Here the coordinator push .xib header or what is his role?

Reply:
it will push another viewController (after it will be created)

Reply:
do you show 2 controllers - tableview and header at the same time?

Reply:
no, it has only one controller ProfileViewController
this delegate is for reacting by tapping on the avatar or edit button. When the user taps, headerViewContent says to its delegate to show SystemProfileViewController (but it doesn't exist yet, that's why we show an alert with "Oops..")

Reply:
ok